src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/config/QueueClientsConfiguration.java

```diff
@@ -13,14 +14,63 @@
 @Profile("!functional & !integration")
 public class QueueClientsConfiguration {
 
+    @Bean
+    @ConfigurationProperties(prefix = "azure.servicebus.payments")
+    private QueueConfig paymentQueueConfig() {
+        return new QueueConfig();
+    }
+
     @Bean
     public IMessageReceiver paymentMessageReceiver(
-        @Value("${azure.servicebus.payments.connection-string}") String connectionString
```

Comment:
`azure.servicebus.payments.connection-string`  can be removed from app yaml  and integration test config ?

Reply:
once everything works - yes